package.json

```diff
@@ -4,7 +4,7 @@
   "description": "prettier plugin for the Ruby programming language",
   "main": "src/ruby.js",
   "scripts": {
-    "build": "pkg --output pkg/prettier --targets=linux,macos,win node_modules/prettier/bin-prettier.js",
+    "build": "pkg --output exe/nodejs/prettier --targets=linux,macos,win node_modules/prettier/bin-prettier.js",
```

Comment:
Changing the destination path to `exe/nodejs` since Rubygems also uses it as the destination path of gem packages.